applications/EasyMultisig.md

```diff
@@ -0,0 +1,102 @@
+# EOS Network Foundation Grant Proposal
+
+- **Project Name:** EasyMultisig
+- **Team Name:** Esteban Saa
+- **EOS Payment Address:** trustswap111
+- **[Level](https://github.com/eosnetworkfoundation/grant-framework#grant-levels):** 1
+- **Pomelo Grant(s):** https://pomelo.io/grants/eosevmmultis
+- Project is Open-Source: Yes
```

Comment:
Please add the asterisk's back to the Open source and token sale questions. Since they are not there, It is breaking the parcer .

Reply:
ok